doc/requirements-rtd.txt

```diff
@@ -6,6 +6,6 @@ pillow==7.2.0
 alabaster>=0.7,<0.8,!=0.7.5
 commonmark==0.8.1
 recommonmark==0.5.0
-sphinx<2
+sphinx==3.0.4
```

Comment:
Got this error when building the docs: https://docs.ray.io/en/latest/development.html#building-the-docs. 

```
ERROR: sphinx-tabs 2.0.1 has requirement sphinx<4,>=2, but you'll have sphinx 1.8.5 which is incompatible.
ERROR: sphinx-book-theme 0.0.39 has requirement docutils>=0.15, but you'll have docutils 0.14 which is incompatible.
```

So I updated the version of the offending sphynx dependency.